x-pack/plugins/siem/public/common/components/exceptions/viewer/exception_entries.tsx

```diff
@@ -0,0 +1,164 @@
+import { transparentize } from 'polished';
+
+import { AndOrBadge } from '../../and_or_badge';
+import { getEmptyValue } from '../helpers';
+import * as i18n from '../translations';
+import { FormattedEntry } from '../types';
+
+const EntriesDetails = styled(EuiFlexItem)`
+  padding: 16px;
+`;
+
+const StyledEditButton = styled(EuiButton)`
+  ${({ theme }) => css`
+    background-color: ${transparentize(0.9, theme.eui.euiColorPrimary)};
+    width: 111px;
+    border: none;
+    font-weight: 600;
+  `}
+`;
+
+const StyledRemoveButton = styled(EuiButton)`
+  ${({ theme }) => css`
+    background-color: ${transparentize(0.9, theme.eui.euiColorDanger)};
+    width: 111px;
+    border: none;
+    font-weight: 600;
+  `}
+`;
+
+const AndOrBadgeContainer = styled(EuiFlexItem)`
+  padding-top: 32px;
+`;
+
+interface ExceptionEntriesComponentProps {
+  entries: FormattedEntry[];
+  handleDelete: () => void;
+  handleEdit: () => void;
+}
+
+const ExceptionEntriesComponent = ({
+  entries,
+  handleDelete,
+  handleEdit,
+}: ExceptionEntriesComponentProps) => {
+  const columns = useMemo(
+    () => [
+      {
+        field: 'fieldName',
+        name: 'Field',
+        sortable: false,
+        truncateText: true,
+        'data-test-subj': 'exceptionFieldNameCell',
+        width: '30%',
+        render: (value: string | null, data: FormattedEntry) => {
+          if (value !== null && data.isNested) {
+            return (
+              <>
+                <EuiIconTip type="grabHorizontal" size="m" />
+                {value}
+              </>
+            );
+          } else {
+            return value ?? getEmptyValue();
+          }
+        },
+      },
+      {
+        field: 'operator',
+        name: 'Operator',
+        sortable: false,
+        truncateText: true,
+        'data-test-subj': 'exceptionFieldOperatorCell',
+        width: '20%',
+        render: (value: string | null) => value ?? getEmptyValue(),
+      },
+      {
+        field: 'value',
+        name: 'Value',
+        sortable: false,
+        truncateText: true,
+        'data-test-subj': 'exceptionFieldValueCell',
+        width: '60%',
+        render: (values: string | string[] | null) => {
+          if (Array.isArray(values)) {
+            return (
+              <EuiFlexGroup direction="row">
+                {values.map((value) => {
+                  return <EuiFlexItem grow={1}>{value}</EuiFlexItem>;
+                })}
+              </EuiFlexGroup>
+            );
+          } else {
+            return values ?? getEmptyValue();
+          }
+        },
+      },
+    ],
+    [entries]
+  );
+
+  return (
+    <EntriesDetails grow={5}>
+      <EuiFlexGroup direction="column" gutterSize="m">
+        <EuiFlexItem>
+          <EuiFlexGroup direction="row" gutterSize="none">
+            {entries.length > 1 && (
+              <AndOrBadgeContainer grow={false}>
+                <AndOrBadge type="and" includeAntenas data-test-subj="exceptionsViewerAndBadge" />
```

Comment:
You have two variations of spelling in your code of antena. You have antenna with two n's and then you have a few spots where it is spelled with just one `n`. I think you want `includeAntennas` with two `nn` as that is what my spell checker is suggesting.

Reply:
Missed that. Updated throughout to use the two `n`s. Thanks for the 👀